build/ci/pr/Jenkinsfile

```diff
@@ -90,3 +104,16 @@ def notOnlyDocs() {
     	returnStatus: true
     ) == 0
 }
+
+def ifImageNeedsRebuilding() {
+    sh """
+        md5sum operators/Gopkg.lock build/ci/Dockerfile local-volume/Gopkg.lock \
+        | awk '{print \$1}' | md5sum | awk '{print \$1}' \
```

Comment:
Why do we need the first `awk`?

```
> md5sum Gopkg.lock ../build/ci/Dockerfile | md5sum | awk '{print $1}' 
ab0d88101895bf53067c4504d3d4d3c9
```

looks good.

Reply:
Answering since I discussed this offline with @artemnikitin:
The first `md5sum` produces an output that contains the relative paths to the files. Depending from where you run this command, the final checksum will differ even though the files content is the same. This is a bit error-prone, hence we strip-out the files location from the first md5sum output.

Reply:
With latest changes we don't need it anyway